docs/src/main/sphinx/release/release-437.md

```diff
@@ -0,0 +1,36 @@
+# Release 437 (24 Jan 2024)
+
+## General
+
+* Add support for `char(n)` values in {func}`to_utf8`. ({issue}`20158`)
+* Add support for `char(n)` values in {func}`lpad`. ({issue}`16907`)
+* {{breaking}} Replace the `exchange.compression-enabled` configuration property
+  and `exchange_compression` session property with
+  [the`exchange.compression-codec`and `exchange_compression_codec` properties](prop-exchange-compression-codec),
+  respectively. ({issue}`20274`)
+* {{breaking}} Replace the `spill-compression-enabled` configuration property 
+  with [the `spill-compression-codec` property](prop-spill-compression-codec). ({issue}`20274`)
+* {{breaking}} Remove the deprecated `experimental.spill-compression-enabled`
+  configuration property. ({issue}`20274`)
+* Fix permanently stale materialized views when using pass-through table
+  functions like `query` and `raw_query`. ({issue}`19904`)
+
+## Delta Lake connector
+
+* Improve query performance for queries that don't use table statistics. ({issue}`20054`)
+
+## Hive connector
+
+* Fix error when coercing union-typed data to a single type when reading Avro
+  files. ({issue}`20310`)
+
+## Iceberg connector
+
+* Improve performance of queries with filters on `ROW` columns stored in Parquet
+  files. ({issue}`17133`)
+
+## SPI
+
+* Fix `ArrayIndexOutOfBoundsException` with RowBlockBuilder during output
+  operations. ({issue}`20426`)
+* Fix adaptation of nullable function return to `BLOCK_BUILDER`. ({issue}`18456`)
```

Comment:
This is not an SPI change. It's just a plain old bug fix. It should be something along the lines of:

* Fix failure when invoking functions that may return null values.